airship/kubeadm-airship-gap-analysis.md

```diff
@@ -25,6 +25,79 @@ Jira issues:
 Jira issues:
 - [Non-default IP for control plane components](https://airship.atlassian.net/browse/AIR-146)
 
+## Motivation
+For this experiment our motivation is to know the answers to the following questions
+
+* How do these components choose which interfaces to use?
+* How do we influence the choice during init phase?
+* How do we influence the choice during join phase?
+* How granular the configuration could be made?
+
+## Experiment Requirements
+In a multi interface control plane node, configuring control plane such that
+
+* IP addresses of control plane components are pre-determined
+* During init phase, control plane components use distinct IP addresses
+* During join phase, control plane components use distinct IP addresses
+
+## Test Cases
+* Apply kubeadm init and join with default configurations to check how it populates the different static pod manifest files
+* Use a custom kubeadm-init config file where we add only api-server related configurations and check if a different interface for api-server is assigned
+* Extend the 2nd test case by adding etcd-server related configurations in kubeadm-init config file and check if different interface for api-server and etcd-server gets assigned.
+* In all of the above cases, consider the impact of these configuration using kubeadm-join config file.
+
+## Test Results
+
+### Test case 1: etcd and api-server IP not given
+**Setup:** No custom kubeconfig-init nor kubeconfig-join provided
```

Comment:
All these are shown as one line in the markdown.
Please add a new line after each bold line

Reply:
the same goes with the other test cases

Reply:
done